src/main/java/com/google/sps/gson/Json.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.gson;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.sps.gson.ChromeOSDeviceDeserializer;
+import com.google.sps.data.ChromeOSDevice;
+
+/*
+ * Class providing the ability to serialize and deserialize Json using Gson and custom-defined
+ * deserializers where appropriate. Should be used by all classes not in this directory when
+ * needing to serialize/deserialize something.
+*/
+public class Json {
```

Comment:
Where is this class used?

Reply:
Right now it is not being used anywhere, this was setup for stuff that we are going to need in the future.  It will be needed to deserialize the request data that we get from the Admin SDK API and serialize some of the data that we send to the user, but these things cannot be implemented until we get OAuth set up.